armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -299,10 +306,12 @@ BITCOIN_VERSION="0.18.0"
 
 mkdir -p /usr/local/src/bitcoin
 cd /usr/local/src/bitcoin/
-curl --retry 5 -SL https://bitcoincore.org/keys/laanwj-releases.asc | gpg --import
+#curl --retry 5 -SL https://bitcoincore.org/keys/laanwj-releases.asc | gpg --import
 curl --retry 5 -SLO https://bitcoincore.org/bin/bitcoin-core-${BITCOIN_VERSION}/SHA256SUMS.asc
 curl --retry 5 -SLO https://bitcoincore.org/bin/bitcoin-core-${BITCOIN_VERSION}/bitcoin-${BITCOIN_VERSION}-aarch64-linux-gnu.tar.gz
 
+## get Bitcoin Core signing key
+gpg --keyserver pool.sks-keyservers.net --recv-keys 01EA5486DE18A882D4C2684590C8019E36C2E964
```

Comment:
(Optional) Another option would be to just add the full PGP key to this repository, thus avoiding any network calls. (GPG keyservers can be notoriously unstable.)

Reply:
I think receiving the key from public server makes it more transparent & trustless (no need to check the actual keyfile). But I can live with both solutions.

Reply:
Okay, let's keep it like this for now.